gateway-service/src/main/java/org/zowe/apiml/gateway/conformance/AbstractSwaggerParser.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.gateway.conformance;
+
+
+import org.springframework.http.HttpMethod;
+import org.zowe.apiml.product.gateway.GatewayConfigProperties;
+
+import java.util.*;
+
+public abstract class AbstractSwaggerParser {
```

Comment:
Just a naming comment, this more like the parser is a validator, right?
I understand it uses the swagger-provided libraries for actual parsing.

Reply:
Renamed